src/fides/api/alembic/migrations/versions/1e750936a141_migrate_to_fideslang_2_0_uses_and_.py

```diff
@@ -0,0 +1,322 @@
+
+        if egress:
+            for item in egress:
+                item["data_categories"] = _replace_matching_data_categories(
+                    item.get("data_categories"), data_label_map
+                )
+
+            update_egress_query: TextClause = text(
+                "UPDATE ctl_systems SET egress = :updated_egress WHERE id= :system_id"
+            )
+            bind.execute(
+                update_egress_query,
+                {"system_id": row["id"], "updated_egress": json.dumps(egress)},
+            )
+
+
+def update_privacy_declaration_data_categories(
+    bind: Connection, data_label_map: Dict[str, str]
+) -> None:
+    """Upgrade or downgrade data uses from fideslang 2.0 for data categories"""
+    existing_privacy_declarations: ResultProxy = bind.execute(
+        text("SELECT id, data_categories FROM privacydeclaration;")
+    )
+    for row in existing_privacy_declarations:
+        labels: Optional[List[str]] = _replace_matching_data_categories(
+            row["data_categories"], data_label_map
+        )
+
+        update_label_query: TextClause = text(
+            "UPDATE privacydeclaration SET data_categories = :updated_label WHERE id= :declaration_id"
+        )
+        bind.execute(
+            update_label_query,
+            {"declaration_id": row["id"], "updated_label": labels},
+        )
+
+
+def update_ctl_policy_data_categories(
+    bind: Connection, data_label_map: Dict[str, str]
+) -> None:
+    """Upgrade or downgrade data categories from fideslang 2.0 for ctl policies"""
+    existing_ctl_policies: ResultProxy = bind.execute(
+        text("SELECT id, rules FROM ctl_policies;")
+    )
+    for row in existing_ctl_policies:
+        rules: List[Dict] = row["rules"]
+        for i, rule in enumerate(rules or []):
+            data_labels: Dict = rule.get("data_categories", {})
+            rule_data_categories: List[str] = data_labels.get("values", [])
+            updated_data_categories: List[str] = _replace_matching_data_categories(
+                rule_data_categories, data_label_map
+            )
+            rules[i]["data_categories"]["values"] = updated_data_categories
+
+        update_data_label_query: TextClause = text(
+            "UPDATE ctl_policies SET rules = :updated_rules WHERE id= :policy_id"
+        )
+        bind.execute(
+            update_data_label_query,
+            {"policy_id": row["id"], "updated_rules": json.dumps(rules)},
+        )
+
+
+def upgrade() -> None:
+    bind: Connection = op.get_bind()
+
+    # Given that our advice is to turn off auto-migrations and make a db copy,
+    # there is no "downgrade" version of this. It also wouldn't be feasible given
+    # it would require an older version of fideslang.
+    logger.info("Removing old default data categories and data uses")
+    bind.execute(text("DELETE FROM ctl_data_uses WHERE is_default = TRUE;"))
+    bind.execute(text("DELETE FROM ctl_data_categories WHERE is_default = TRUE;"))
+
```

Comment:
So we have them upgrade, then they have to restart fides again to get the seed script to run, are those the right steps?